sticker-utils/src/subcommands/train.rs

```diff
@@ -170,22 +176,24 @@ impl TrainApp {
         let mut best_epoch = 0;
         let mut best_acc = 0.0;
         let mut last_acc = 0.0;
+        let global_step = 0;
 
         let mut lr_schedule = self.lr_schedule();
         let mut save_scheduler = self.save_schedule.to_save_scheduler("");
 
         for epoch in 0.. {
-            let lr = lr_schedule.learning_rate(epoch, last_acc);
+            let lr = lr_schedule.compute_epoch_learning_rate(epoch, last_acc);
 
-            let (loss, acc) = self.run_epoch(
+            let (loss, acc, global_step) = self.run_epoch(
```

Comment:
This global_step gets dropped at the end of the scope, leading to a learning rate that resets after every epoch.

Reply:
Whoops! Also explains why the global_step above didn't need to be `mut`. Easy to fix though ;).

Reply:
Maybe we should also start showing the current LR in the progress, besides the loss?

Reply:
Tensorboard :). But I agree, it's also nice to see it without resorting to a browser.